bosh/releases/pre_render_scripts/diego-cell/sle15-rootfs-setup/jobs/patch_pre-start.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Could you do the patching following the refactor I did on https://github.com/SUSE/kubecf/pull/261?